war/src/main/webapp/help/system-config/systemMessage.html

```diff
@@ -1,5 +1,6 @@
 <div>
-  This message will be displayed on <a href=".">the top page</a>.
-  Useful for posting a system-wide notification to users.
-  Can contain HTML tags or whatever markup language is defined for the system.
+  This message will be displayed at the top of the
```

Comment:
I'm not sure what (if anything) should be changed here — the message seems clear enough to me.

The message is shown at the top of the main page — not literally at the very top, but under the header and navbar that sit above virtually every page.

The message is also shown at the top of views (good to know, thanks), but not necessarily *every* view, depending on whether it's a dashboard view or something else — but then again, those would come from a plugin.